.github/ISSUE_TEMPLATE/BUG_REPORT.md

```diff
@@ -32,5 +32,5 @@ Please provide the version number where this issue was encountered.
 ## Checklist
 
 <!-- TODO: Update the link below to point to your project's contributing guidelines -->
```

Comment:
I think we can remove the TODO here and in the PULL_REQUEST_TEMPLATE since you've updated the links, no?